conda/base/context.py

```diff
@@ -1094,10 +1094,10 @@ def os_distribution_name_version(self):
         #   'Windows', '10.0.17134'
         platform_name = self.platform_system_release[0]
         if platform_name == "Linux":
-            from .._vendor.distro import id, version
-
             try:
-                distinfo = id(), version(best=True)
+                import distro
```

Comment:
just realized that we did the appdirs -> platformdirs deprecation wrong, we need to include the fallback to the vendored code otherwise we'll have issues with packaging (chicken and egg problem)

```suggestion
                try:
                    import distro
                except ImportError:
                    from .._vendor import distro
```

Reply:
Good point!